src/components/SessionEventContainer/SessionEventContainer.js

```diff
@@ -250,6 +250,7 @@ const SessionEventContainer = ({ history, queryClient }) => {
     // We only want to configure the event listeners once, not every time
     // there is a change to stripes or history. Hence, an empty dependency
     // array.
+    // should `stripes.rtr` go here?
   }, []); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
@zburke can you comment on this? As-is, this configuration cannot be modified at runtime by `ui-developer`.

Reply:
Probably I started with a non-empty array and it didn't work as expected, so I added the empty array as a crutch and never looked back. Your observation/criticism is fair, but I don't have time to debug this further since the only thing "broken" about it is that it can't be modified at runtime via dev-tools.

Reply:
Tested it, and it works beautifully. Only the RTR config in ui-developer causes the effect to re-run, nothing else (not even the other config forms in developer)

Reply:
It does rely on not calling `configureRtr` on render in `Root`, though.